spec/jobs/release_object_job_spec.rb

```diff
@@ -3,7 +3,6 @@
 require 'rails_helper'
 
 RSpec.describe ReleaseObjectJob do
```

Comment:
Note that the tests changed a lot less that it appears from the diffs.  I basically added two separate contexts: 'with already published objects' and 'when objects have never been published'.  The `Dor::Workflow::Client` instance then mocks the return value to verify the publish doesn't happen.  The nesting of the existing tests in a new context block and the addition of the new mock for `Dor::Workflow::Client` resulted in a messy diff.  The end state is really just a few new tests.